components/BackupAndRestoreBannerNotification.tsx

```diff
@@ -65,7 +67,9 @@ export const BackupAndRestoreBannerNotification: React.FC = () => {
           testId={'restoreBackupSuccess'}
         />
       )}
-      {restoreController.isBackUpRestoreFailure && restoreFailure()}
+      {restoreController.isBackUpRestoreFailure &&
+        !restoreController.isDownloadBackupFileFromCloud &&
```

Comment:
Same comment here as well, 
Is the condition - !restoreController.isDownloadBackupFileFromCloud redundant here? as isBackUpRestoreFailure itself would give us failure status?

Reply:
resolved